### DISCUSSION TITLE
Representante da APyB para assinar com patrocinador.

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#164] [Pergunta]
Bom dia, pessoal! Tudo bem?
Recebemos o seguinte email do patrocinador "Instituto de Pesquisas Eldorado":
![image](https://github.com/user-attachments/assets/9fa5fe9d-805c-498d-b79d-8f5223a39030)

Gostaria de saber se teria como alguém da diretoria da APyB pudesse estar assinando como representante da Python Norte 2024 para darmos continuidade ao processo. Caso seja possível, por favor, devolcer com as seguintes informações:
a.           Nome Completo: 
b.           CPF: 
c.            E-mail: 
d.            Certificado digital (ICP-Brasil/e-cpf): SIM ou NÃO:

Pode ser para o [nosso email oficial](mailto:norte@python.org.br) que encaminhamos para o patrocinador.
Muito obrigado!

cc @anicelysantos

**Comment by anicelysantos** [comment]
Oie @MateusSantosMeuBem 
Enviei um email agora com cópia para você com as informações.
